Trustroots/trustroots
# Convert "reference thread" feature into React

#### Proposed Changes

* Convert "reference thread" feature into React
* Put it back to message thread. It went missing when messages were converted to React

#### Testing Instructions

* Open message thread when there are no messages; you should not see anything
    <img width="1213" alt="Screenshot 2020-12-27 at 13 11 13" src="https://user-images.githubusercontent.com/87168/103170892-891e9b00-4850-11eb-93a3-ba4cab7e26c6.png">

* Write a message to someone, and refresh the thread. You should still see nothing.

* Only when someone else writes you, you should see the question
    <img width="1218" alt="Screenshot 2020-12-27 at 13 11 47" src="https://user-images.githubusercontent.com/87168/103170902-9b98d480-4850-11eb-98f6-02e98db6dcdd.png">

* Response stays visible. Refresh the thread to test it.
    <img width="481" alt="Screenshot 2020-12-27 at 13 47 17" src="https://user-images.githubusercontent.com/87168/103170926-bbc89380-4850-11eb-9fb8-b57130ab42b2.png">
    <img width="472" alt="Screenshot 2020-12-27 at 13 47 13" src="https://user-images.githubusercontent.com/87168/103170930-bcf9c080-4850-11eb-94ab-cdcacd48eb2c.png">

* You can change your response by pressing "change"
